.github/workflows/cli.yaml

```diff
@@ -0,0 +1,37 @@
+name: CLI Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
```

Comment:
Missing timeout

Reply:
I don't think this project needs to set a global timeout for CI, the open source project is not limited by the CI quota for the owner account (currently it is free) and it has a maximum timeout of 6 hours.

But I think it is necessary to add a timeout to the next tmate debugger, if it is not limited it will keep running until the CI timeout.